CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+## [Unreleased]
+### Added
+- Added enwiki feature lists
```

Comment:
- Add polarity score https://github.com/wikimedia/draftquality/commit/d30655953a92e0234b72584a1bc5cfb15f25ac44
- Add words_to_watch to enwiki feature list https://github.com/wikimedia/draftquality/commit/458cdfa055dc814a90a31db0c468c880f17a41b7

Reply:
updated!